src/__test__/iotx.integration.test.ts

```diff
@@ -108,9 +108,70 @@ accountTest("deployContract", async t => {
   const hash = await antenna.iotx.deployContract({
     from: creator.address,
     amount: "0",
-    data: contract.bytecode,
+    data: Buffer.from(contract.bytecode, "hex"),
     gasLimit: "1000000"
   });
 
   t.truthy(hash);
 });
+
+accountTest("executeContract", async t => {
```

Comment:
I am feeling `executeContract ` and `readResultContract ` are not high level enough 

can we make it as simple as 

```
await contract.methods.set(666);
const value = await contract.methods.get();
t.deepEqual(value, '666');
```

?